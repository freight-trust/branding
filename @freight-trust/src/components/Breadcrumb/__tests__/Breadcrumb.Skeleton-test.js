/**
 * Copyright IBM Corp. 2016, 2018
 *
 * This source code is licensed under the Apache-2.0 license found in the
 * LICENSE file in the root directory of this source tree.
 */

import React from 'react';
import { BreadcrumbSkeleton } from '../';
import { mount } from 'enzyme';

describe('BreadcrumbSkeleton', () => {
  it('should render', () => {
    const wrapper = mount(<BreadcrumbSkeleton />);
    expect(wrapper).toMatchSnapshot();
  });
});
