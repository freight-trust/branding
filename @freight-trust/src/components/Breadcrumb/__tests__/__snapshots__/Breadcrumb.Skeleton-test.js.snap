// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BreadcrumbSkeleton should render 1`] = `
<BreadcrumbSkeleton>
  <div
    className="bx--breadcrumb bx--skeleton"
  >
    <div
      className="bx--breadcrumb-item"
    >
      <span
        className="bx--link"
      >
         
      </span>
    </div>
    <div
      className="bx--breadcrumb-item"
    >
      <span
        className="bx--link"
      >
         
      </span>
    </div>
    <div
      className="bx--breadcrumb-item"
    >
      <span
        className="bx--link"
      >
         
      </span>
    </div>
  </div>
</BreadcrumbSkeleton>
`;
